### DISCUSSION TITLE
Trabajo Practico 8

### DISCUSSION THREAD

**Discussion by martinvilu** [INGCOM-UNRN-P1/cursada-2023#199] [Avisos]
https://classroom.github.com/a/Rpy0eRgT

Fecha de entrega: 19 de octubre

Fecha límite:  26 de octubre

**Comment by Nikzza** [comment]
Martín en el campus bimodal el TP8 que enviaste para el repositorio lo mandaste mal, te manda directamente al tp7

**Reply by martinvilu** [reply]
Corregido
